index.html

```diff
@@ -49,6 +49,16 @@
 <tr>
 <td>
 
+<p id="datetime"></p>
+<script>
+// Get current date and time
+var now = new Date();
+var datetime = now.toLocaleString();
+
+// Insert date and time into HTML
+document.getElementById("datetime").innerHTML = datetime;
```

Comment:
```suggestion
document.getElementById("datetime").innerHTML = new Date(document.lastModified);
```
De datum van het laden van de pagina werd gebruikt in plaats van de datum van de laatste wijziging.